### PR TITLE
Fixed blog title content.

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -79,20 +79,20 @@ module.exports = function(language, isStaging) {
 
   var categories = {
     'showcase': {
-      name: 'Showcase',
-      title: 'Showcase of apps made with Onsen UI and Monaca'
+      name: 'Showcases',
```
Comment (argelius) [inline config.js]: I think just "Showcase" makes more sense

Comment (andipavllo) [inline config.js]: Thanks @argelius , I'll edit it. What do you think about Tutorials instead of Tutorial?

Comment (argelius) [inline config.js]: I think "Tutorials" is good

Comment (argelius) [inline config.js]: @andipavllo

**Pull request merged**